src/components/ex-navigation/ExNavigation.scss

```diff
@@ -0,0 +1,118 @@
+.navigation {
+  width: 20% !important;
```

Comment:
There is one media query that does not yet show the burger, but the search is pushed to the next row. It is not a catastrophe, but maybe you can cover this case somehow.

![image](https://user-images.githubusercontent.com/88790097/134134237-ebb6519b-072a-40e0-a91c-2841d17da02c.png)

Reply:
fixed media query

Reply:
Menu items and search should be in a row:
![image](https://user-images.githubusercontent.com/88790097/134165527-4e3d80f3-9474-4cf4-8567-389ef5551a2d.png)